src/NumericalAlgorithms/SphericalHarmonics/Strahlkorper.hpp

```diff
@@ -88,18 +89,51 @@ class Strahlkorper {
                         std::array<double, 3> center);
 
   /// Prolong or restrict another surface to the given `l_max` and `m_max`.
+  template <typename OtherFrame>
```

Comment:
The `Strahlkorper{coefs, other_strahlkorper}` achieves that if you use the `other_strahlkorper.coefficients()` for the first argument. I don't really want to deal with the template gymnastics I'll have to do to avoid a conflict between the regular copy constructor, and a "copy constructor" for a different frame.

Reply:
```c++
Strahlkorper(const Strahlkorper&) = default;
Strahlkorper(Strahlkorper&&) = default;

template <typename OtherFrame>
Strahlkorper(const Strahlkorper<OtherFrame>& other) : /* ... */ {}
```
?

(Probably `explicit`.)